webpack/assets/javascripts/react_app/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history';
+
+export default createBrowserHistory();
```

Comment:
Why add this file instead of directly importing it in `ReactApp.js`?

Reply:
* We might want to import it from different files.
* We might want to add some extra configuration here.